core/new-gui/src/app/common/service/storage.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class StorageService {
+  constructor() {
```

Comment:
why do we need this as a service? these functions can just be static utils functions

Reply:
I just don't find a good place to put those static functions. Any suggestions?

Reply:
put under `common/util/storage.ts`